x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curations.tsx

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+
+import { useValues, useActions } from 'kea';
+
+import {
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiPageContent,
+  EuiTitle,
+  EuiBasicTable,
+  EuiBasicTableColumn,
+  EuiEmptyPrompt,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { FlashMessages } from '../../../../shared/flash_messages';
+import { KibanaLogic } from '../../../../shared/kibana';
+import { Loading } from '../../../../shared/loading';
+import { EuiButtonTo, EuiLinkTo } from '../../../../shared/react_router_helpers';
+import { ENGINE_CURATIONS_NEW_PATH, ENGINE_CURATION_PATH } from '../../../routes';
+import { generateEnginePath } from '../../engine';
+
+import { CURATIONS_OVERVIEW_TITLE, CREATE_NEW_CURATION_TITLE } from '../constants';
+import { CurationsLogic } from '../curations_logic';
+import { Curation } from '../types';
+
+export const Curations: React.FC = () => {
+  const { dataLoading, curations, meta } = useValues(CurationsLogic);
+  const { loadCurations } = useActions(CurationsLogic);
+
+  useEffect(() => {
+    loadCurations();
+  }, [meta.page.current]);
+
+  if (dataLoading && !curations.length) return <Loading />;
+
+  return (
+    <>
+      <EuiPageHeader>
+        <EuiPageHeaderSection>
+          <EuiTitle size="l">
+            <h1>{CURATIONS_OVERVIEW_TITLE}</h1>
+          </EuiTitle>
+        </EuiPageHeaderSection>
+        <EuiPageHeaderSection>
+          <EuiButtonTo to={generateEnginePath(ENGINE_CURATIONS_NEW_PATH)} fill>
+            {CREATE_NEW_CURATION_TITLE}
+          </EuiButtonTo>
+        </EuiPageHeaderSection>
+      </EuiPageHeader>
+      <EuiPageContent>
+        <FlashMessages />
+        <CurationsTable />
+      </EuiPageContent>
+    </>
+  );
+};
+
+export const CurationsTable: React.FC = () => {
+  const { dataLoading, curations, meta } = useValues(CurationsLogic);
+  const { onPaginate, deleteCurationSet } = useActions(CurationsLogic);
+
+  const columns: Array<EuiBasicTableColumn<Curation>> = [
+    {
+      field: 'queries',
+      name: i18n.translate(
+        'xpack.enterpriseSearch.appSearch.engine.curations.table.column.queries',
```

Comment:
I noticed that some of these messages are not typed according to the [i18n guidelines](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types).

It may be that these don't fit into one of those categories, and it doesn't appear to me that there is any guidance on how to handle it when it doesn't fit into one of those types.

It may also be that we may not be concerned about this at this point.

What's your take on this, should we find more appropriate names?

Reply:
I didn't see any guidance for tables - lmk if I missed it?

I do see one `'kbn.management.editIndexPattern.scripted.table.nameDescription'` example which I think this mostly fits. We could probably change it to `queriesTitle` instead of just `queries`, but I also think the current ID string is fairly descriptive 🤷